guest-agent/VfioService/VfioService/Program.cs

```diff
@@ -18,10 +18,14 @@ static void Main()
             using (var manager = new ClientManager(form))
             {
                 form.ClientManager = manager;
+                InterceptMouse.ClientManager = manager;
+                InterceptMouse.Start();
                 manager.SendCommand(CommandOut.ReportBoot);
 
                 var _ = form.Handle; // create form without showing
+                
                 Application.Run();
+                InterceptMouse.Stop();
```

Comment:
Doesn't `Application.Run` return if you end the process via the task manager? Doesn't that try to just close the window first?